spec/changelog.yml

```diff
@@ -10,6 +10,9 @@
     - description: Add support for semantic_text field definition.
       type: enhancement
       link: https://github.com/elastic/package-spec/pull/807
+    - description: Add support for next steps links to kibana manifest.
+      type: enhancement
+      link: https://github.com/elastic/package-spec/pull/834
```

Comment:
@jsoriano do you think this feature should be just available starting in spec 3.4.0 ? These links would not be shown in Kibana until the support is added.

Would it be interesting to add a JSON patch to remove the new `links` field for versions before `3.4.0`?

Reply:
> @jsoriano do you think this feature should be just available starting in spec 3.4.0 ? These links would not be shown in Kibana until the support is added.

Does it break anything in current versions of the stack?
If the only problem is that links will not be rendered, I would say to stay in 3.3.
If something breaks before adapting kibana, then we need to use 3.4.

> Would it be interesting to add a JSON patch to remove the new links field for versions before 3.4.0?

Yes, I would add this for the version we decide at the end.

Reply:
> Does it break anything in current versions of the stack?
>If the only problem is that links will not be rendered, I would say to stay in 3.3.
>If something breaks before adapting kibana, then we need to use 3.4.

I think that it shouldn't break anything. The "worst" that can happen is that those links get ignored by Kibana, as it reads only those keys that we explicitly choose. So we could stay in 3.3 for this change.